src/Compression/CompressionCodecQATZSTD.cpp

```diff
@@ -0,0 +1,114 @@
+#ifdef ENABLE_QATZSTD_COMPRESSION
+#include <Compression/CompressionInfo.h>
+#include <Compression/CompressionFactory.h>
+#include <zstd.h>
+#include <Parsers/IAST.h>
+#include <Parsers/ASTLiteral.h>
+#include <Parsers/ASTFunction.h>
+#include <Common/typeid_cast.h>
+#include <Poco/Logger.h>
+#include <Common/logger_useful.h>
+#include "qatseqprod.h"
+#include "CompressionCodecZSTD.h"
```

Comment:
--> `#include <Compression/CompressionCodecZSTD.h>` + please move that up to the other Compression includes.

Reply:
Done